src/landscape.cpp

```diff
@@ -1306,14 +1306,12 @@ void OnTick_LinkGraph();
 
 void CallLandscapeTick()
 {
-	{
-		PerformanceAccumulator framerate(PFE_GL_LANDSCAPE);
+	PerformanceAccumulator framerate(PFE_GL_LANDSCAPE);
 
-		OnTick_Town();
-		OnTick_Trees();
-		OnTick_Station();
-		OnTick_Industry();
-	}
+	OnTick_Town();
+	OnTick_Trees();
+	OnTick_Station();
+	OnTick_Industry();
 
```

Comment:
Removing {} changes the scope of the PerformanceAccumulator to the whole function, instead of just the requested block.

Reply:
Thanks. I'll keep the {} then.